### DISCUSSION TITLE
请问有没有办法在修改函数名并deploy后删除旧的函数？

### DISCUSSION THREAD

**Discussion by ron-liu** [Serverless-Devs/Serverless-Devs#399] [Q&A] [answered]
重现步骤：
1. 创建名为hello的函数，并部署
2. 在`s.yml`中修改hello为hello1，并在此部署
3. 创建了新函数`hello1`，但原来的`hello`函数还在。

**Comment by heimanba** [comment, marked as answer]
工具不会主动帮您删除资源，您可以在部署后执行。s remove 进行删除。

**Reply by ron-liu** [reply]
谢谢
